examples/workflows/call_failing_activity_workflow.rb

```diff
@@ -0,0 +1,14 @@
+require 'activities/failing_activity'
+
+# exercises a workflow's ability to handle errors in user code, which get
+# re-raised in the workflow.
+class CallFailingActivityWorkflow < Temporal::Workflow
+  def execute(message)
+    begin
+      FailingActivity.execute!(message)
+    rescue => e
+      return {class: e.class, message: e.message}
```

Comment:
Why not just re-raise the error? I'm assuming it will expose the same issue and is a bit clearer

Reply:
Agreed.  Once my other PR is in, it'll be easier to test that, as it's a bit difficult to extract the exception from the payload.  I can hold off til that's in.

Reply:
See comment above.